presto-client/src/main/java/com/facebook/presto/client/IntervalDayTime.java

```diff
@@ -55,4 +63,30 @@ public static String formatMillis(long millis)
 
         return format("%s%d %02d:%02d:%02d.%03d", sign, day, hour, minute, second, millis);
     }
+
+    public static long parseMillis(String value)
+    {
+        if (value.equals(LONG_MIN_VALUE)) {
```

Comment:
So the parser only handles the exact format of the format with no extra whitespace... maybe document that